src/main/java/Task.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Task object to describe a task to be stored in task list.
+ */
+public class Task {
+
+    private String description;
+    protected boolean completed;
+    protected int completedCode;
+
+    /**
+     * Creates a task object.
+     * @param description description of the task.
+     */
+    public Task(String description) {
+
+        this.description = description;
+        this.completed = false;
+        this.completedCode = 0;
+
+    }
+
+    /**
+     * Changes the status of completed of the task, if method is invoked, completed will be turn to true.
```

Comment:
Slight grammatical errors in this JavaDoc comment :) Also, the boolean naming should be something like isCompleted (you can refer to [this page](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html) for the coding standard for naming of booleans!)
```suggestion
     * Changes the status of completion of the task. If the method is invoked, isCompleted will be turned to true.
```